swagger.json

```diff
@@ -4042,6 +4042,69 @@
                     "description": "ID of the group if the organization has multiple groups (uncommon).",
                     "example": 101
                 },
+                "timezone": {
+                    "type": "string",
+                    "description": "Timezone used by the driver for daily logs (e.g. America/New_York, Europe/Paris), if different from the organization default.",
+                    "example": "America/New_York"
+                },
+                "carrierNameOverride": {
+                    "type": "string",
+                    "description": "Carrier’s name, if different from the organization default.",
+                    "example": "Samsara"
+                },
+                "mainOfficeAddressOverride": {
+                    "type": "string",
+                    "description": "Carrier’s office address, if different from the organization default.",
+                    "example": "444 De Haro St, San Francisco"
+                },
+                "carrierUsDotNumberOverride": {
+                    "type": "integer",
+                    "description": "Carrier’s US DOT number (max: 999999999), if different from the organization default.",
+                    "example": "999999999"
+                },
+                "tags": {
+                    "type": "array",
+                    "description": "An array of tags that the driver belongs to.",
+                    "items": {
+                        "type": "string",
+                        "example": "ExampleTag"
```

Comment:
It might be better to put a better example tag here maybe like a tag that ArcBest currently uses?